src/main/java/cz/mzk/fedora/model/DataStreams.java

```diff
@@ -0,0 +1,38 @@
+package cz.mzk.fedora.model;
+
+import java.util.Optional;
+
+public enum DataStreams {
+
+    DC("DC", "text/xml", "false", "X", "A"),
+    RELS_EXT("RELS-EXT", "application/rdf+xml" , "true", "X", "A");
+//
+
+    public String name;
+    public String mimeType;
+    public String versionable;
+    public String controlGroup;
+    public String state;
+
+
```

Comment:
Možná ne vždy to jde, ale snaž se mezi logickými bloky koda dodržovat pouze jeden řádek.